development/development.env

```diff
@@ -7,8 +7,6 @@ NAUTOBOT_BANNER_TOP="Local"
 NAUTOBOT_CHANGELOG_RETENTION=0
 
 NAUTOBOT_DEBUG=True
-NAUTOBOT_DJANGO_EXTENSIONS_ENABLED=True
```

Comment:
I'd like to see these as togglable items but can be considered out of scope for now

Reply:
`NAUTOBOT_DEBUG` toggles that.